k8s/apis/k8s/v1alpha1/webhook_test.go

```diff
@@ -138,12 +133,12 @@ var _ = BeforeSuite(func() {
 		}
 		conn.Close()
 		return nil
-	}).Should(Succeed())
-}, 60)
+	})
+	assert.NoError(t, err)
 
-var _ = AfterSuite(func() {
+	time.Sleep(1 * time.Second)
```

Comment:
Move the sleep after the cancel() to avoid some errors such as the webhook server routine is not exited.